docs/feeds.md

```diff
@@ -14,41 +14,39 @@ In the *Hooks* pattern, we set up a feed using a [webhook](https://en.wikipedia.
 <!-- The github feed is implemented using webhooks.  Put a link here when we have the open repo ready -->
```

Comment:
We should add link to github package in catalog repo
- https://github.com/openwhisk/openwhisk-catalog/blob/master/packages/github/webhook.js